### ISSUE TITLE
Some traders are not selectable after logout/login

### ISSUE THREAD

**Issue by ghost** [CitizensDev/Citizens#313]
Hey there,

i have a poblem with the traders. Everything works fine, BUT after some serverrestarts they are not selectable anymore. The do talk to me (talk-when-close). I had to select them via "npc select xy" move them out of their current location and then move back. 
No Errors in the console.

**Comment by ghost** [comment]
okay...new  fault localisation:
when i logout near a treader and the login again, traders arent selectable anymore. When i run away from the npc (~15-30 blocks), logout/login, then they are normal again. This is for every user different -depending on "where" they login.

 Any help/solution?

**Comment by ghost** [comment]
seems to be not fixed in the new Citizensversion

**Comment by fullwall** [comment]
Are there phantom npcs in the way?

**Comment by ghost** [comment]
no, they are talking to me and look at me, but are not selectable anymore and i cant walk through them like they are air. I tested this only for traders so far. I can select them via /npc select command and rename them or something else. then they work correctly again. the problem is, that this happens only for people who logged out near them and logged in later.

**Comment by ghost** [comment]
wrong button -.-
